jupyterlab_templates/extension.py

```diff
@@ -105,11 +146,16 @@ def load_jupyter_server_extension(nb_server_app):
         nb_server_app (NotebookWebApplication): handle to the Notebook webserver instance.
     """
     web_app = nb_server_app.web_app
-    template_dirs = nb_server_app.config.get("JupyterLabTemplates", {}).get(
-        "template_dirs", []
-    )
+    jupyterlab_templates_config = nb_server_app.config.get("JupyterLabTemplates", {})
+    nb_server_app.log.info("jupyterlab_templates config: %s" % jupyterlab_templates_config)
+
+    template_dirs = jupyterlab_templates_config.get("template_dirs", [])
+    if isinstance(template_dirs, str):
+        template_dirs = template_dirs.split(",")
 
-    if nb_server_app.config.get("JupyterLabTemplates", {}).get("include_default", True):
+    include_default = jupyterlab_templates_config.get("include_default", True)
+    include_default = include_default.lower() == 'true' if isinstance(include_default, str) else include_default
```

Comment:
When the configs are sent using arguments, it's always set as string, so we need to convert to bool.